scope-validation-announcement.md

```diff
@@ -0,0 +1,10 @@
+# We are keep improving our service! But we might affect you , please read this.
+With the initial release of the ACS we allowed to request the scopes for the tokens. Please refer here for the [Access Token scopes](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-javascript)
```

Comment:
nit: "With the initial release of the ACS, we allowed to request specific scopes for the user access tokens, to achieve a specific ACS functionality.."